labs/src/main/scala/org/scalalabs/basic/lab03/OptionExercise.scala

```diff
@@ -37,6 +42,10 @@ object OptionExercise02 {
    * to convert a possible numeric String (e.g. Some("12")) to an integer
    */
   def totalPeopleInRooms(rooms: Map[Int, Option[String]]): Int = {
-    error("Fix me")
+    rooms.foldLeft(0)((total, room) => total + (room._2 match {
+      case Some(i) if (i == "locked") => 0
+      case Some((i:String)) => i.toInt
+      case _=> 0
+    }))
```

Comment:
Rather than something like `Some(i) if (i == "locked")`, you can also write `Some("locked")`. It's convenient for constant values. Unfortunately, I'm not sure how to do that, or if you can, if `"locked"` was stored in some variable. Because then the match would just overwrite that variable.